README.md

```diff
@@ -7,7 +7,7 @@ Publisher is sometimes referred to as "mainstream publisher".
 - [Completed transaction](https://www.gov.uk/done/make-lpa)
 - [Guide](https://www.gov.uk/council-tax-appeals)
 - [Help page](https://www.gov.uk/help/accessibility)
-- [Licence](https://www.gov.uk/day-nurseries-wales)
+<!-- - [Licence](https://www.gov.uk/day-nurseries-wales) -->
```

Comment:
I think this could just be deleted (if for no other reason than aesthetically it breaks the list rendering in markdown slightly).